lib/api/apiUtils/object/lifecycle.js

```diff
@@ -151,9 +154,37 @@ function getLocationConstraintErrorMessage(locationName) {
         `- ${locationName} - is not listed in the locationConstraint config`;
 }
 
+/**
+ * validateMaxScannedEntries - Validates and returns the maximum scanned entries value.
+ *
+ * @param {object} params - Query parameters
+ * @param {object} config - CloudServer configuration
+ * @param {number} min - Minimum number of entries to be scanned
```

Comment:
we don't have arbitrary values here : actually only two values (1 or 3), with a significant explanation for the "3":
> 3 is required as a minimum because we must scan at least three entries to determine version eligibility.
> Two entries representing the master key and the following one representing the non-current version.

So it seems to me we actually want to embed this knowledge in the function, with a much simpler parameter (like `currentOnly` ?)